src/components/table/partial-styles/tabulator-paginator.scss

```diff
@@ -1,6 +1,17 @@
 @use '../../../style/mixins';
 @use '../../../style/functions';
 
+#tabulator-container {
+    &:not(.has-pagination) {
+        // When there is no pagination, Tabulator still displays the pagination bar,
+        // but with only 1 page in it. This is why we hide it:
+        .tabulator-paginator {
+            visibility: hidden;
+            display: none !important;
+        }
+    }
+}
+
```

Comment:
the 4th commit fixes this issue: 

<img width="776" height="387" alt="Image" src="https://github.com/user-attachments/assets/8ab1796a-0cc5-4311-8acf-4fbc001f0008" />